config/rbac/role.yaml

```diff
@@ -59,6 +59,18 @@ rules:
   - patch
   - update
   - watch
+- apiGroups:
+  - control.k8ssandra.io
+  resources:
+  - cassandratasks
+  verbs:
+  - create
+  - delete
+  - get
+  - list
+  - patch
```

Comment:
Are patch/update needed by k8ssandra-operator?

Reply:
As of right now, they are not needed.